log/log.go

```diff
@@ -66,6 +63,11 @@ func Get() *logrus.Logger {
 	default:
 		log.Level = logrus.InfoLevel
 	}
+
+	rawLog.Formatter = new(RawFormatter)
+}
+
+func Get() *logrus.Logger {
```

Comment:
Why are we changing this here?

Reply:
The modification in `log.Level` from Get is a race condition which was triggered by tests. I moved the modification into the init() function, another option would be to wrap it with a sync.Once.

Reply:
Moved out.